examples/streetlights-operation-security.yml

```
@@ -60,14 +68,16 @@ channels:
       message:
         $ref: '#/components/messages/turnOnOff'
       security:
-      # This operation level security implies the ability to send a message to 
+      # This operation level security implies the ability to subscribe messages 
```

WDYT about the following 2 rewordings:

```suggestion
      # This operation level security implies the ability to subscribe to messages from
```

```suggestion
      # `smartylighting.streetlights.1.0.action.{streetlightId}.turn.on` channel with Authorization headers 
```